### PR TITLE
add backgroundImage and boxShadow into tailwind.config.js and comments on syntax to use them in tailwind css

### LINKED ISSUE #18: Add the colors and fonts design pattern to the tailwind.config.
Go to figma Design System and add the fonts and colors to the tailwind.config.

### PR DESCRIPTION
**Pull request by Takkunkuma**
close #18 

#18 was closed once but reopened and put into review section due to modification on the color, boxshadow parts.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -2,4 +2,29 @@
...
+    }
+    h2 {
+
+    }
+    h3 {
+
+    }
+    h4 {
+        
+    }
```
Comment (BenjaminnHuang) [inline src/index.css]: Remove this if not used.

**Review by BenjaminnHuang — COMMENTED** [review]
Need some edit

File: src/index.css (hunk 1 above)
Comment (Takkunkuma) [reply src/index.css]: oops

**Review by BenjaminnHuang — APPROVED** [review]
LGTM!

**Pull request merged**